vinyldns.go

```diff
@@ -142,6 +142,58 @@ func main() {
 				},
 			},
 		},
+		{
+			Name:        "zone-create",
+			Usage:       "zone-create --name <name> --email <email> --admin-group-id <adminGroupID> --transfer-connection-name <transferConnectionName> --transfer-connection-key <transferConnectionKey> --transfer-connection-key-name <transferConnectionKeyName> --transfer-connection-primary-server <transferConnectionPrimaryServer> --zone-connection-name <zoneConnectionName> --zone-connection-key <zoneConnectionKey> --zone-connection-key-name <zoneConnectionKeyName> --zone-connection-primary-server <zoneConnectionPrimaryServer>",
+			Description: "Create a zone",
+			Action:      zoneCreate,
+			Flags: []cli.Flag{
+				cli.StringFlag{
+					Name:  "name",
+					Usage: "The zone name",
+				},
+				cli.StringFlag{
+					Name:  "email",
+					Usage: "The zone email",
+				},
+				cli.StringFlag{
+					Name:  "admin-group-id",
+					Usage: "The zone admin group ID",
+				},
+				cli.StringFlag{
+					Name:  "transfer-connection-name",
+					Usage: "The zone transfer connection",
+				},
+				cli.StringFlag{
+					Name:  "transfer-connection-key-name",
+					Usage: "The zone transfer connection key name",
+				},
+				cli.StringFlag{
+					Name:  "transfer-connection-key",
+					Usage: "The zone transfer connection key",
+				},
+				cli.StringFlag{
+					Name:  "transfer-connection-primary-server",
+					Usage: "The zone transfer connection primary server",
+				},
+				cli.StringFlag{
+					Name:  "zone-connection-name",
+					Usage: "The zone connection",
+				},
+				cli.StringFlag{
+					Name:  "zone-connection-key-name",
+					Usage: "The zone connection key name",
+				},
+				cli.StringFlag{
+					Name:  "zone-connection-key",
+					Usage: "The zone connection key",
+				},
+				cli.StringFlag{
+					Name:  "zone-connection-primary-server",
```

Comment:
on thing missing from the zone info is the ACL rules.  That is probably a ton though from a CLI standpoint.  Worth noting, but I wouldn't add it here.  Fact is, ACL rules are not widely used.

Reply:
Are ACL rules declared during zone creation? Or are they something that are only used in reading the zone?

Reply:
They _can_ be declared on zone creation.  I would argue though that they are really cumbersome from a cli / ux perspective and should be omitted.

Reply:
Alrighty; let's punt on ACL rules with regard to zone creation, though I [made an issue](https://github.com/vinyldns/vinyldns-cli/issues/28) surrounding displaying them via the `zone` command.